src/rred_reports/reports/emails.py

```diff
@@ -0,0 +1,83 @@
+"""Emailing of reports to users"""
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+import pytz
+from exchangelib import Account, FileAttachment, Mailbox, Message
+
+from rred_reports.reports.auth import RREDAuthenticator
+
+
+@dataclass
+class EmailContent:
+    """Email content class"""
+
+    account: Account
+    recipients: list[str]
+    cc_recipients: Optional[list[str]]
+    subject: str
+    body: str
+    attachment: Optional[bytes]
+    attachment_filename: Optional[str]
+
+
+@dataclass
```

Comment:
May be missing this but is the dataclass aspect of this used?

Reply:
Not any more, no! Will remove that decorator